mistralrs-server/src/main.rs

```diff
@@ -92,6 +92,14 @@ struct Args {
     /// Number of prefix caches to hold on the device. Other caches are evicted to the CPU based on a LRU strategy.
     #[arg(long, default_value_t = 16)]
     prefix_cache_n: usize,
+
+    /// Number of prompt completions to run concurrently in prompt mode.
+    #[clap(long, default_value_t = 1)]
+    prompt_concurrency: usize,
+
+    /// Number of prompt tokens to generate.
+    #[clap(long, default_value_t = 128)]
+    prompt_max_tokens: usize,
```

Comment:
Should these be a sub-command of sorts? I feel like they could confuse someone.

These are only used in prompt mode. I change these settings a lot when profiling

Reply:
Yes, if you could make this a sub command that would be great!

Reply:
Clap doesn't allow 2 sub-commands, so I made the prompt arguments require prompt, and improved docs.